Tasks/ChefPowerShell/ChefPowerShell.ps1

```diff
@@ -0,0 +1,44 @@
+[CmdletBinding(DefaultParameterSetName = 'None')]
+param
+(
+    [String] [Parameter(Mandatory = $true)]
+    $ConnectedServiceName,
+
+    [String] [Parameter(Mandatory = $true)]
+    $ScriptPath,
+
+    [String] [Parameter(Mandatory = $false)]
+    $ScriptArguments
+)
+
+Write-Host "Entering script ChefPowerShell.ps1"
+
+Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.Chef"
```

Comment:
Change this module name to Microsoft.TeamFoundation.DistributedTask.Task.Deployment.Chef